docs/source/gmaps.rst

```diff
@@ -221,7 +221,9 @@ Markers are currently limited to the Google maps style drop icon. If you need to
 
     gmaps.configure(api_key="AI...")
 
-    starbucks_locations = gmaps.datasets.load_dataset("starbucks_uk")
+    starbucks_kfc_locations = gmaps.datasets.load_dataset("starbucks_kfc_uk")
+    starbucks_locations = [(i[0], i[1]) for i in starbucks_kfc_locations if i[2] == 'starbucks'] 
+
     starbucks_layer = gmaps.symbol_layer(
         starbucks_locations, fill_color="green", stroke_color="green", scale=2
     )
```

Comment:
Sure, that is indeed my case. I used the query method and also needed to drop the chain_name column.